### PR TITLE
[FEATURE 💡] Remove Nike from header in mobile devices enhancement

### LINKED ISSUE #67: [FEATURE 💡] Remove Nike from header in mobile devices
### Description

- Remove the nike text from the logo **(Only for mobile devices)**
- Make the navbar a bit shorter for **(Only for mobile devices)**
- The logo and icons must also be made smaller 
- The theme icon seems not aligned properly
- Take reference from the original site webpage screenshots given below

### Screenshots

image

- Reference
image

## Happy contributing. 💐
Star my other Repositories [here](https://github.com/warmachine028)

### PR DESCRIPTION
**Pull request by sahilrw**
# Issue #67  📐

- I [Sahil Wankhade](https://github.com/sahilrw) have worked for #67 

## Guidelines 🔐

I accept the fact that I have followed the guidelines and have not copied the codes from around the internet.

- [x] **Contribution Guidelines**
- [x] **Code of Conduct**

## Issue to be closed 🛅

- My pull request closes #67

## Screenshots 📷

Here are the pictures of changes that I have made 🔽

image
image

  
  
  
---

### CODE REVIEW

**Comment by warmachine028** [comment]
@sahilrw please look into the PR template and make corrections in that

**Review by warmachine028 — COMMENTED** [review]
Please look into the changes asked

**Review by warmachine028 — CHANGES_REQUESTED** [review]
- @sahilrw please look into the PR template and make corrections in that

- Resolve your merge conflicts by pulling the latest branch

**Comment by warmachine028** [comment]
Navbar has become thinner also on desktop devices

**Comment by sahilrw** [comment]
https://github.com/warmachine028/nike/pull/69/commits/df6016ae284c2584fe3fc3464733c38095b1dd84
Pushed the changes.

**Comment by warmachine028** [comment]
> [df6016a](https://github.com/warmachine028/nike/commit/df6016ae284c2584fe3fc3464733c38095b1dd84) Pushed the changes.

Please pull the latest changes from main to resolve the merge conflicts

**Comment by sahilrw** [comment]
Completed the PR template.
Merged the conflicts.

File: src/components/Navbar.jsx
Comment (warmachine028) [inline src/components/Navbar.jsx:1]: There are errors in this file, please rectify them
How did you resolve the conflicts?

image

**Review by warmachine028 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**